src/Microsoft.DotNet.Cli.Utils/FileAccessRetryer.cs

```diff
@@ -47,5 +48,35 @@ public static class FileAccessRetrier
                 }
             }
         }
+
+        /// <summary>
+        /// Run Directory.Move and File.Move in Windows has a chance to get IOException with
```

Comment:
@peterhuene an better idea to convey this only work for X.Move?

@nguerrera since we know this is the indexer now. I added indexer to the comment

Reply:
I'm fine with the comment.